src/components/ActionMenu/components/SecondaryAction/SecondaryAction.scss

```diff
@@ -9,6 +9,7 @@ $button-spacing: rem(12px);
   button {
     @include focus-ring($border-width: rem(1px));
     background: transparent !important;
+    color: var(--p-text) !important;
```

Comment:
This is kind of weird as it makes the button look not destructive. Should we remove the ability for destructive secondary actions?

Reply:
Can primary be destructive still? If so then I think we should remove primary from the allowed props here

Reply:
There are instances where the secondary action is destructive. Pretty sure I've need that.

Reply:
@alex-page this is what you're referring to, right? The "View" action which is secondary doesn't have a differentiating colour?

<img width="1602" alt="Screen Shot 2021-01-13 at 4 43 13 PM" src="https://user-images.githubusercontent.com/8923697/104605956-251c1500-564d-11eb-9c66-78ffdef30fd2.png">

And seconding @dleroux - I've needed a destructive secondary action in `ActionMenu` on several occasions

Reply:
Yeah exactly the `View` button looks like the other secondary actions. We might need to get some additional UX opinions but I wonder if the balance a destructive button in the page would bring too much attention, if that is the case should it look the same or is there a way to make it look destructive without it being so distracting.